client/src/pages/About.tsx

```diff
@@ -1,8 +1,231 @@
+import jodiPic from '../imgs/jodiPic.png';
+
+const Box_Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+  margin-top: 1900px;
+  gap: 15px;
+`;
+
+const FlexBoxWrapper = styled.div`
+  height: 85vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 100px;
+  margin-right: 50px;
+`;
+
+const ProjectBox = styled.div`
+  width: 999px;
+  height: 850px;
+  background-color: ${COLORS.PURPLE_LIGHT};
+  border-radius: 22px;
+  margin-top: 30px;
+  margin: 5px;
+  text-align: center;
+`;
+
+const Box = styled.div`
+  width: 999px;
+  height: 400px;
+  background-color: ${COLORS.PURPLE_LIGHT};
+  border-radius: 22px;
+  margin-top: 30px;
+  margin: 5px;
+  text-align: center;
+`;
+
+const WordContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  border-radius: 25px;
+  gap: 20px;
+  margin-top: 25px;
+`;
+
+const PageTitle = styled(H1)`
+  color: ${COLORS.WHITE};
+  position: relative;
+  text-align: center;
+`;
+const SmallHeading = styled(SubTitle)`
+  font-size: 5rem;
+`;
+
+const BoxText = styled(P)`
+  padding: 10px;
+  color: ${COLORS.BLURPLE};
+  text-align: center;
+  font-size: 2rem;
+`;
+
+const ParagraphText = styled(P)`
+  padding: 20px;
+  color: ${COLORS.BLURPLE};
+  text-align: left;
+  font-size: 2rem;
+`;
+
+const AboutUs = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  margin-left: 20px;
+  margin-top: -5px;
+`;
+const Img = styled.img`
+  align-items: center;
+  width: 290px;
+  height: 325px;
+  border-radius: 8px;
+`;
 
 function About() {
-  return <PageWrapper pageTitle="About"></PageWrapper>;
+  return (
+    <PageWrapper pageTitle="About">
+      <FlexBoxWrapper>
+        <Box_Wrapper>
+          <WordContainer>
+            <PageTitle>About Us</PageTitle>
+          </WordContainer>
+          <WordContainer>
+            <SmallHeading>
+              <br />
+              Our Project
+            </SmallHeading>
+          </WordContainer>
+          <ProjectBox>
+            <h1>
+              <br />
+              <br /> <img src={Logo} alt="OuterWhorld" width="400" />
+            </h1>
+            <BoxText>
+              <br /> Computer Science and Engineering - University of Nevada,
+              Reno
+            </BoxText>
+            <BoxText>
+              <br />
+              Team 5: Erin E. Marden, Andrei N. Iorgulescu, Jodi A. Hieronymus
+            </BoxText>
+            <BoxText>
+              <br />
+              Instructed by David Feil-Seifer, Devrin Lee
+            </BoxText>
+            <BoxText>
+              <br />
+              Advised by Erin Keith
+            </BoxText>
+            <ParagraphText>
+              <br />
+              <br />
+              OuterWhorld is going to be an interactive web-based application
```

Comment:
This paragraph needs to be changed from future tense to present tense or we're gonna get scolded most likely. I think Project 4 had a present tense paragraph!